src/calendar/index.js

```diff
@@ -297,12 +299,12 @@ class Calendar extends Component {
   }
 
   render() {
-    const {enableSwipeMonths, style} = this.props;
+    const {enableSwipeMonths, style, gestureStyle} = this.props;
     const GestureComponent = enableSwipeMonths ? GestureRecognizer : View;
     const gestureProps = enableSwipeMonths ? {onSwipe: (direction, state) => this.onSwipe(direction, state)} : {};
 
     return (
-      <GestureComponent {...gestureProps}>
+      <GestureComponent {...gestureProps} style={gestureStyle} >
```

Comment:
Why passing the style to the GestureComponent and not to the View below it using the 'style' prop?

Reply:
It seemed when testing that all intermediary containers had to also have flex style to behave as expected. E.g. 

```
    <View style={[g.flex, g.backgroundBlue]}>
      <View style={[g.backgroundRed]}>
        <View style={[ g.flex, g.backgroundYellow]} />
      </View>
    </View>
```
results in 
<img width="355" alt="Screen Shot 2021-04-22 at 2 07 45 PM" src="https://user-images.githubusercontent.com/7525998/115785475-5c007600-a374-11eb-9ab1-0c98f50a2518.png">

and it is only when I add the flex style to the middle view that I see what I expect:
```
    <View style={[g.flex, g.backgroundBlue]}>
      <View style={[g.flex, g.backgroundRed]}>
        <View style={[ g.flex, g.backgroundYellow]} />
      </View>
    </View>
```
<img width="366" alt="Screen Shot 2021-04-22 at 2 11 24 PM" src="https://user-images.githubusercontent.com/7525998/115785696-a5e95c00-a374-11eb-9d6a-69a9932886c9.png">

Reply:
Ok. But I'll change the prop name to `containerStyle` as this is the top-level container, while `guesterStyle` is understood only by someone that is familiar with the source code.